src/aics-image-viewer/components/App/index.tsx

```diff
@@ -576,6 +544,11 @@ const App: React.FC<AppProps> = (props) => {
     return () => window.removeEventListener("resize", onResizeDebounced);
   }, []);
 
+  // one-time init after view3d exists and before we start loading images
+  useEffect(() => {
+    view3d.setCameraMode(viewerSettings.viewMode);
+  });
+
```

Comment:
This can also be accomplished by converting the regular `setCameraMode` hook to a `useImageLoadEffect`. Either approach is good, though here I believe you also need to give `useEffect` an empty dependency array to make it run once on load rather than on every render.

Reply:
oh yeah, I'll add the empty array.  I'd like to see this be the one place where other global init on view3d might happen as we expand the options.  Anyway, for now it's simple enough.

I thought useImageLoadEffect depended on there being an image. The problem this is fixing is that the view was staying in 3d until after the image data arrives.

Reply:
`useImageLoadEffect` (which is distinct from `useImageEffect`) will also run immediately on image load, without waiting for a React render cycle, to prevent flashes like this. The way you have it might still be a better approach for this case though, since this property isn't reset by image loads. Load effects are mainly properties like brightness, render mode, interpolation etc. which are owned by `VolumeDrawable` and need to be resynced when the current volume is replaced.